quickstarts/dotnet/c#/config.yml

```diff
@@ -0,0 +1,68 @@
+id: 
```

Comment:
```suggestion
```

Reply:
I have seen that, even if the "id" is left blank, the preview worked. 👍

Reply:
Also, I have removed the "id" field and then pushed my changes 👍